lib/internal/bootstrap/switches/is_not_main_thread.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const { ObjectDefineProperty } = primordials;
+const rawMethods = internalBinding('process_methods');
+const {
+  unavailable
+} = require('internal/process/worker_thread_only');
+
+process.abort = unavailable('process.abort()');
+process.chdir = unavailable('process.chdir()');
+process.umask = wrappedUmask;
+process.cwd = rawMethods.cwd;
+
+delete process._debugProcess;
+delete process._debugEnd;
+delete process._startProfilerIdleNotifier;
+delete process._stopProfilerIdleNotifier;
+
+function defineStream(name, getter) {
+  ObjectDefineProperty(process, name, {
+    configurable: true,
+    enumerable: true,
+    get: getter
+  });
+}
+
+defineStream('stdout', getStdout);
+defineStream('stdin', getStdin);
+defineStream('stderr', getStderr);
+
+// Worker threads don't receive signals.
+const {
+  startListeningIfSignal,
+  stopListeningIfSignal
+} = require('internal/process/signal');
+process.removeListener('newListener', startListeningIfSignal);
```

Comment:
IIUC we don't support listening signals from non-main thread right now.

Reply:
Yes, the intention was that it looks symmetrical when you put the `is_not_main_thread.js` and `is_main_thread.js` side-by-side - even though this would be essentially a no-op (the delete calls above were added for the same purpose). I could add some comments if they look confusing.

Reply:
hmm, on the other hand, maybe this part and the deletion should be moved to `does_{not}_own_process_state` at this point? @addaleax

Reply:
@legendecas Is this a blocking comment?

Reply:
Sorry, it's `removeListener` 😅. Marking as resolved.